job_templates/sag_pt_model_learner/config_fed_client.conf

```diff
@@ -0,0 +1,34 @@
+format_version = 2
+executors = [
+    {
+        # tasks that the defined Executor will support
+        tasks = [
+            # training task
+            "train"
+            # cross-site validation tasks
+            "submit_model"
+            "validate"
+        ]
+        executor {
+            id = "Executor"
+            path = "nvflare.app_common.executors.model_learner_executor.ModelLearnerExecutor"
+            args {
+                # id must match the id of the ModelLearner component
+                learner_id = "cifar10-learner"
+            }
+        }
+    }
+]
+task_result_filters = []
+task_data_filters = []
+components = [
+    {
+        id = "cifar10-learner"
+        path = "fl.model_learner.CIFAR10ModelLearner"
```

Comment:
if we copy this into custom (by using job cli), we can use `path = "model_learner.CIFAR10ModelLearner"`